### PR TITLE
Completed html-forms exercise

### PR DESCRIPTION
**Pull request by jeff-cruz**
Kapture 2022-03-15 at 12 45 43

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <p>Email
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: This text should be a `label` element

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <input type="text" name="username" size="20">
+        </p>
+      </div>
+
+      <div>
+        <p>Why is coding cool?</p>
```
Comment (codingcodymiller) [inline html-forms/index.html:16]: This text should be a `label` element

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+      <input type="radio" name="language" value="HTML" /> HTML
+      <input type="radio" name="language" value="CSS" /> CSS
+      <input type="radio" name="language" value="JavaScript" /> JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:30]: This text next to each radio input should be a `label` element

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+      <div>
+        <p>Would you like to sign up for more information?</p>
+      </div>
+      <input type="checkbox" name="check" value="signup" /> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: This "Sign me up!" text should be a `label` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by jeff-cruz** [comment]
Screen Shot 2022-03-15 at 1 04 46 PM

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email</label>
+      </div>
+
+      <input type="text" name="username" size="20">
```
Comment (codingcodymiller) [inline html-forms/index.html:13]: The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+      <div>
+        <label for="cool">Why is coding cool?</label>
+      </div>
+      <select name="option">
```
Comment (codingcodymiller) [inline html-forms/index.html:18]: The value of the `id` attribute on the `select` needs to match the value of the `for` attribute on the `label` it corresponds with.

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+
+      <input type="radio" name="language" value="HTML" />
```
Comment (codingcodymiller) [inline html-forms/index.html:29]: The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.

File: html-forms/index.html [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+      <label for="HTML">HTML</label>
+
+      <input type="radio" name="language" value="CSS" />
```
Comment (codingcodymiller) [inline html-forms/index.html:32]: The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**